### PR TITLE
Use jsonc to parse nls manifest

### LINKED ISSUE #833: Error parsing JSON manifest translations file
Hi,

I have faced a small issue while packaging my extension with _translations_. The error was fairly easy to find, but it was a surprise  since no error occurred while developing (debugging) the extension.

My `package.nls.json` file had an extra comma, at the end of the file:

```json
{
    . . . 
    "pascal.commands.whatsNewContextMenu.title": "What's New",
    "pascal.commands.installDelphiThemes.title": "Pascal: Install Delphi Themes", // <- this one
}
```

I totally understand the JSON is not well formed, but I think it should be a consistent behavior between the extension development/debugging and the packaging process. Maybe the same validation should be made while debugging.

Thank you

### PR DESCRIPTION
**Pull request by joaomoreno**
fixes #833

### CODE REVIEW

File: src/test/fixtures/nls/package.json [hunk 1]
```diff
@@ -4,6 +4,8 @@
 	"version": "1.6.2",
 	"publisher": "ms-vscode",
 	"description": "%extension.description%",
+	
+	// These are categories
```
Comment (ulugbekna) [inline src/test/fixtures/nls/package.json:8]: is this intentional?

Comment (joaomoreno) [reply src/test/fixtures/nls/package.json:8]:
```suggestion
```

Definitely not!

Comment (joaomoreno) [inline src/test/fixtures/nls/package.json:8]:
```suggestion
```

Comment (joaomoreno) [reply src/test/fixtures/nls/package.json:8]: Definitely not, thanks for the heads up!

**Review by ulugbekna — APPROVED** [review]

**Pull request merged**